docs/source/inspect.rst

```diff
@@ -29,14 +29,17 @@ objects
 
 .. code-block:: python
 
-   >>> (x + 1).dask
+   >>> dict((x + 1).dask)
    {('wrapped_1', 0, 0): (ones, (5, 5)),
     ('wrapped_1', 0, 1): (ones, (5, 5)),
     ('wrapped_1', 0, 2): (ones, (5, 5))
     ('x_1', 0, 0): (add, ('wrapped_1', 0, 0), 1),
     ('x_1', 0, 1): (add, ('wrapped_1', 0, 1), 1),
     ('x_1', 0, 2): (add, ('wrapped_1', 0, 2), 1)}
 
+.. note:: Here we've kept the names like ``x_1``, ``ones``, and ``add`` simple
```

Comment:
"simple" -> "simply"

Reply:
The intent here is actually "simple" .  I've rephrased things to remove this ambiguity.